packages/browser-wallet/src/popup/pages/AddAccount/i18n/en.ts

```diff
@@ -1,5 +1,5 @@
 const t = {
-    chooseIdentity: 'Which identity should create the account?',
+    chooseIdentity: 'Which identity do you want to use to create the account?',
```

Comment:
We need some padding on the element, where this is used.

Reply:
And text center as well.

Reply:
nice catch!